Explore the label settings dialog description (#4212)
qgis/QGIS-Documentation: source/docs/user_manual/style_library/label_settings.rst

   Text settings - Text tab

> In the |text| :guilabel:`Text` tab, you can set for each label:

Reviewer: ```suggestion
In the |text| :guilabel:`Text` tab, you can set:
```

* the :guilabel:`Font`, from the ones available on your machine

> * the :guilabel:`Style`: along the common styles of the font, you can set

Reviewer: ```suggestion
* the :guilabel:`Style`: along with the common styles of the font, you can set
```

>   whether the text should be underlined or strikeout

Reviewer: ```suggestion
  whether the text should be underlined or striked through
```

* the :guilabel:`Size` that can be in :guilabel:`Points`, :guilabel:`Pixels`,

>   :guilabel:`Millimeter`, :guilabel:`Map units`, :guilabel:`Meters at scale`,

Reviewer: ```suggestion
  :guilabel:`Millimeters`, :guilabel:`Map units`, :guilabel:`Meters at scale`,
```

formats stored within your :ref:`style manager database <vector_style_manager>`.
This allows you to easily set text formats to match styles saved in the local
style database, and also to add a new text format to the style database based on
the current settings. Press the :guilabel:`Save format...` button to store the
current text format in the :guilabel:`Style Manager`, providing a name and tag(s).

Likewise, a label settings list widget is shown when configuring labels, allowing
you to pick from the |styleManager| :sup:`Style Manager` widget or to add new
styles to it.

.. _labels_formatting:

Formatting tab
--------------

.. _figure_textsettings_formatting:

.. figure:: img/textsettings_formatting.png
   :align: center

   Text settings - Formatting tab

In the |labelformatting| :guilabel:`Formatting` tab, you can:

* Use the :guilabel:`Type case` option to change the capitalization style of
  the text. You have the possibility to render the text as :guilabel:`All
  uppercase`, :guilabel:`All lowercase` or :guilabel:`Capitalize first letter`.
  Note that the last option modifies only the first letter of each word and

>   leaves untouched the other letters in the text.

Reviewer: ```suggestion
  leaves the other letters in the text untouched.
```

* Under :guilabel:`Spacing`, change the space between words and between
  individual letters.
* |checkbox| :guilabel:`Enable kerning` of the text font
* Set the :guilabel:`Text orientation` which can be :guilabel:`Horizontal`
  or :guilabel:`Vertical`. It can also be :guilabel:`Rotation-based` when
  setting a label.
* Use the :guilabel:`Blend mode` option to determine how your labels will mix
  with the map features below them (more details at :ref:`blend-modes`).

> * The |unchecked| :guilabel:`Apply label text substitutes` option gives you

Reviewer: ```suggestion
* The |unchecked| :guilabel:`Apply label text substitutes` option makes you
```

>   ability to specify a list of texts to substitute to texts in feature labels

Reviewer: ```suggestion
  able to specify a list of texts to substitute to texts in feature labels
```

>   (e.g., abbreviating street types). Replacement texts are thus used to display

Reviewer: ```suggestion
  (e.g., abbreviating street types). Replacement texts are used when displaying
```

>   labels in the map canvas. Users can also export and import lists of

Reviewer: ```suggestion
  labels on the map canvas. Users can also export and import lists of
```

  substitutes to make reuse and sharing easier.

> * Configure :guilabel:`Multiple lines` behavior:

Reviewer: ```suggestion
* Configure :guilabel:`Multiple lines`:
```

>   * Set a character for a line break in the text with the :guilabel:`Wrap on

Reviewer: ```suggestion
  * Set a character that will force a line break in the text with the :guilabel:`Wrap on
```

    character` option

>   * Set an auto-wrapping ideal line size using the :guilabel:`Wrap lines to`

Reviewer: ```suggestion
  * Set an ideal line size for auto-wrapping using the :guilabel:`Wrap lines to`
```

    option. The size can represent either the :guilabel:`Maximum line length`
    or the :guilabel:`Minimum line length`.

>   * Decide of the :guilabel:`Line Height`

Reviewer: ```suggestion
  * Decide the :guilabel:`Line Height`
```

  * Format the :guilabel:`Alignment`: typical values available are
    :guilabel:`Left`, :guilabel:`Right` and :guilabel:`Center`.

    When setting point labels properties, the text alignment can also be
    :guilabel:`Follow label placement`. In that case, the alignment will depend
    on the final placement of the label relative to the point. E.g., if the
    label is placed to the left of the point, then the label will be right
    aligned, while if it is placed to the right, it will be left aligned.

* For line labels you can include :guilabel:`Line direction symbol`

>   to help determine the lines directions, with symbols to use to indicate the

Reviewer: ```suggestion
  to help determine the line directions, with symbols to use to indicate the
```

  :guilabel:`Left` or :guilabel:`Right`. They work particularly well when
  used with the *curved* or *Parallel* placement options from the
  :guilabel:`Placement` tab. There are options to set the symbols position, and
  to |unchecked| :guilabel:`Reverse direction`.
* Use the |unchecked| :guilabel:`Formatted numbers` option to format numeric
  texts. You can set the number of :guilabel:`Decimal places`. By default, ``3``
  decimal places will be used. Use the |checkbox| :guilabel:`Show plus sign` if

>   you want to show the plus sign in positive numbers.

Reviewer: ```suggestion
  you want to show the plus sign for positive numbers.
```

.. _labels_buffer:

Buffer tab
----------

.. _figure_textsettings_buffer:

.. figure:: img/textsettings_buffer.png
   :align: center

   Text settings - Buffer tab

> To create a buffer around the labels, activate the |checkbox| :guilabel:`Draw

Reviewer: ```suggestion
To create buffers around the labels, activate the |checkbox| :guilabel:`Draw
```

Reviewer: Also OK to use singular.

text buffer` checkbox in the |labelbuffer| :guilabel:`Buffer` tab. Then you can:

* Set the buffer's :guilabel:`Size` that can be in :guilabel:`Points`,

>   :guilabel:`Pixels`, :guilabel:`Millimeter`, :guilabel:`Map units`,

Reviewer: ```suggestion
  :guilabel:`Pixels`, :guilabel:`Millimeters`, :guilabel:`Map units`,
```
